src/waltham/waltham-private.h

```diff
@@ -157,4 +157,21 @@ wth_object_new_with_id(struct wth_connection *connection, uint32_t id);
 struct wth_object *
 wth_object_new(struct wth_connection *connection);
 
+/** Send an error on a specific object
+ *
+ * \param conn The Waltham connection.
+ * \param obj A Waltham object.
+ * \param code The error code.
+ * \param str The error message.
+ *
+ * This function wraps the wth_display_send_error display function, so
+ * the server doesn’t have to use this private API.
+ *
+ * \memberof wth_connection
+ * \server_api
+ */
+void
+wth_connection_send_error(struct wth_connection *conn, struct wth_object *obj,
+                          uint32_t code, const char *str);
+
```

Comment:
This is good, but is the `WL_EXPORT` still there in the function definition? That should go as well.

Reply:
Ah, that was done already.

Reply:
I already removed that `WL_EXPORT` in a previous patch, this one is just moving the forward declaration to a different file.